specification/attributes/amortization_handling.md

```diff
@@ -0,0 +1,34 @@
+# Amortization Handling
+
+Amortization is a process used to break down and spread purchase costs over the period of time or term of use. 
+
+Amortization is primarily used in purchasable commitment-based discounts. When a purchase is applicable to resources, like commitment-based discounts, the amortized cost of a resource takes the initial payment and term into account and distributes it out based on resource usage, attributing the prorated cost for each hour of billing. Amortization provides a method for FinOps practitioners to reallocate significant purchase charges to the appropriate audience in support of their cost allocation efforts. 
+
+## Attribute ID
+
+AmortizationHandling
+
+## Attribute name
+
+Amortization Handling
+
+## Description
+
+Rules and requirements defining how amortization is shown in the billing data.
+
+## Requirements
+
+* Every charge MUST include all applicable discounts. (e.g., negotiated and commitment-based discounts)
+* Discounts SHOULD be applied directly to the charges they are discounting.
+* All purchases SHOULD be amortized.
+* MUST NOT add extra rows to subtract the costs due to an applied discount.
```

Comment:
Maybe "discount" should be refined with "discount related to a purchase".

Indeed, GCP SUD is a discount, but it's not a related to a purchase

Reply:
I believe this is covered in the latest version. Please take a look and leave any comments. Resolving this one, since this reference is gone now.